testsuite/domain/src/test/resources/host-configs/host-slave.xml

```diff
@@ -141,6 +141,8 @@
         </server>
         <server name="reload-two" group="reload-test-group"  auto-start="false">
             <jvm name="default" />
+            <!-- this is for ServerRestartRequiredTestCase.testSocketBinding() unless the socket-binding-group is actually used for something it is ignored on the slaves, this is a quick fix before correcting the test -->
+            <socket-bindings socket-binding-group="reload-sockets" port-offset="1550"/>
```

Comment:
Yep, opened https://issues.jboss.org/plugins/servlet/mobile#issue/WFCORE-982
for this test.

I think we understand the issue though. The sbg is unused on the slave, so
it ignores until its actually used for something. Am I missing something
there in how that should work?
On Sep 16, 2015 10:35 AM, "Brian Stansberry" <notifications@github.com>
wrote:

> In testsuite/domain/src/test/resources/host-configs/host-slave.xml
> <https://github.com/wildfly/wildfly-core/pull/1070#discussion_r39645582>:
>
> > @@ -141,6 +141,8 @@
> >          </server>
> >          <server name="reload-two" group="reload-test-group"  auto-start="false">
> >              <jvm name="default" />
> > +            <!-- this is for ServerRestartRequiredTestCase.testSocketBinding() unless the socket-binding-group is actually used for something it is ignored on the slaves, this is a quick fix before correcting the test -->
> > +            <socket-bindings socket-binding-group="reload-sockets" port-offset="1550"/>
>
> We need to correct this problem as part of this work. Unless my
> understanding of what is going on with this test is incorrect, this isn't a
> case of a incorrecdt test, it's the software itself that is not working
> correctly.
>
> AIUI, in the domain config this test uses (domain-standard.xml), the
> "reload-sockets" s-b-g is used by the "reload-test-group" server group.
> Therefore this "reload-two" server-config means the "reload-sockets" is
> used by this server even without this element.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wildfly/wildfly-core/pull/1070/files#r39645582>.
>